docs/scos/dev/back-end-development/data-manipulation/data-interaction/search/search.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Data Manipulation
+description: how to work with Search in the Spryker SCOS back end
+template: concept-topic-template
+related:
+  - title: Facet filter overview and configuration 
+  - link: docs/scos/dev/back-end-development/data-manipulation/data-interaction/search/facet-filter-overview-and-configuration.html
+  - title: Configure search features
+  - link: docs/scos/dev/back-end-development/data-manipulation/data-interaction/search/configure-search-features.html
+  - title: Configure Elasticsearch
+  - link: docs/scos/dev/back-end-development/data-manipulation/data-interaction/search/configure-elasticsearch.html
+  - title: Configure a search query
+  - link: docs/scos/dev/back-end-development/data-manipulation/data-interaction/search/configure-a-search-query.html
+  - title: Configure Search for multi-currency 
+  - link: docs/scos/dev/back-end-development/data-manipulation/data-interaction/search/configure-search-for-multi-currency.html
+  - title: Expand search data
+  - link: docs/scos/dev/back-end-development/data-manipulation/data-interaction/search/expand-search-data.html
+---
```

Comment:
do we really need the related articles here and in other landing pages below?

Reply:
I changed the related articles in the landing pages to links on the topic page (as we discussed after you reviewed a previous PR)